styled/resources.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components/native';
+import { PoppinsText } from './shared';
```

Comment:
This is deprecated - import from `BaseComponents` if you want to extend for text